code/modules/clothing/factions/konyang.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/clothing/under/konyang
+	name = "hanbok"
+	desc = "A traditional Konyang hanbok."
+	desc_fluff = "An ancient form of dress carried over from Earth by the initial Konyang colonists; the hanbok has since become a symbol of Konyang and its most iconic formalwear."
+	icon_state = "hanbok"
+	worn_state = "hanbok"
+	item_state = "hanbok"
+	slot_flags = SLOT_OCLOTHING | SLOT_ICLOTHING
+
+/obj/item/clothing/under/konyang/pink
```

Comment:
Simce this is the path of the previous hanbok you don't have to copypaste repeated things (e.g. the description)